app/locale/en.js

```diff
@@ -1635,15 +1635,15 @@ module.exports = {
       bobby_title: 'Senior Game Designer',
       brian_title: 'Senior Game Design Manager',
       stephanie_title: 'Customer Support Specialist',
-      sdr_title: 'Sales Development Representative',
+      sdr_title: 'Customer Success Manager',
```

Comment:
we dont need to change sdr title, it is correct. We should use `csm_title` instead for Noah 😅  since he switched role.